pkg/rollupsmachine/cartesimachine/server.go

```diff
@@ -0,0 +1,126 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package cartesimachine
+
+import (
+	"fmt"
+	"io"
+	"log/slog"
+	"os/exec"
+	"regexp"
+	"strconv"
+
+	"github.com/cartesi/rollups-node/internal/linewriter"
```

Comment:
Move `linewriter` to `pkg`

Reply:
We only put "exported libraries" in `pkg`.